.travis.yml

```diff
@@ -0,0 +1,9 @@
+sudo: false
+language: ruby
+cache:
+  directories:
+    - vendor/bundle
+    - .gradle
+rvm:
+  - jruby-1.7.24
+script: rake bootstrap test:install-default && rake test:core test:plugins
```

Comment:
@purbon I noticed that the ci setup removes the 'vendor' directory. Do we really want this for every test run? That will really slow down the specs! Maybe we can discuss tomorrow?